src/heuristicblocking.js

```diff
@@ -211,6 +211,13 @@ HeuristicBlocker.prototype = {
       snitch_map.setItem(tracker_origin, firstParties);
     }
 
+    var action_map = this.storage.getBadgerStorageObject('action_map');
+    // If tracker_origin is not yet in action_map, add it with action `constants.ALLOW`
+    if (!action_map.hasItem(tracker_origin)) {
```

Comment:
Fixed in e7f396b613a01dc8a9096d5cc088a41deb04a29f. @alexristich Do we even need this check anymore, given that we debounced `_syncStorage`?